src/KeyedFrames.jl

```diff
@@ -161,6 +161,24 @@ function delete!(kf::KeyedFrame, ind::Vector{<:Symbol})
     return kf
 end
 
+##### RENAME #####
+
+function rename!(kf::KeyedFrame, nms)
+    rename!(kf.frame, nms)
+
+    for (from, to) in nms
+        kf.key[findfirst(kf.key, from)] = to
```

Comment:
I guess you'll want to throw an appropriate error if `from` doesn't exist (`findfirst` will return 0 in that condition).

Reply:
Of course. I was a fool to not run tests locally first. Ugh, this makes the code a lot uglier.

Reply:
Meh, it's still a pretty simple/short function.